snews_pt/snews_pt_utils.py

```diff
@@ -423,3 +427,110 @@ def get_name():
 
     """
     return os.getenv("DETECTOR_NAME")
+
+
+def is_snews_format(snews_message, is_test=False):
```

Comment:
`is_test` is never passed. Test scenarios are therefore not distinguished and can no longer be executed.
in cs I overcame this by checking for `this is a test` in the meta field 
[here](https://github.com/SNEWS2/SNEWS_Coincidence_System/blob/d69ce87ec52025f1f7a7668625a6e3f4c7ce6bea/snews_cs/cs_utils.py#L230)
and [here](https://github.com/SNEWS2/SNEWS_Publishing_Tools/blob/6c7d5bd0368c231cd13d5f8a884f7f9769277769/snews_pt/auxiliary/try_scenarios.py#L29) they are appended to every test message

Reply:
instead of taking `is_test` as an argument (in which case we would need to pass around another `is_test` argument around within snews_pub) I'd say do a similar parsing within this function, and decide whether this is a test message or someone really sent a message with wrong input (e.g. neutrino times from future)

Reply:
I changed the testing check to meta data like with snews-cs